packages/replay/jest.setup.ts

```diff
@@ -71,20 +77,13 @@ const toHaveSameSession = function (received: jest.Mocked<ReplayContainer>, expe
   };
 };
 
-/**
- * Checks the last call to `fetch` and ensures a replay was uploaded by
- * checking the `fetch()` request's body.
- */
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-const toHaveSentReplay = function (
-  _received: jest.Mocked<ReplayContainer>,
-  expected?: SentReplayExpected | { sample: SentReplayExpected; inverse: boolean },
-) {
-  const { calls } = (getCurrentHub().getClient()?.getTransport()?.send as MockTransport).mock;
-  const lastCall = calls[calls.length - 1]?.[0];
+type Result = {passed: boolean, key: string, expectedVal:SentReplayExpected[keyof SentReplayExpected], actualVal: SentReplayExpected[keyof SentReplayExpected]};
+type Call = [EnvelopeHeader, [[ReplayEventHeader|undefined, ReplayEventPayload|undefined], [RecordingHeader|undefined, RecordingPayloadHeader|undefined]]];
+type CheckCallForSentReplayResult = {pass: boolean, call: Call|undefined, results: Result[]}
```

Comment:
looks like the linter didn't get it's hands on this

```suggestion
type Result = {passed: boolean, key: string, expectedVal: SentReplayExpected[keyof SentReplayExpected], actualVal: SentReplayExpected[keyof SentReplayExpected]};
type Call = [EnvelopeHeader, [[ReplayEventHeader|undefined, ReplayEventPayload|undefined], [RecordingHeader|undefined, RecordingPayloadHeader|undefined]]];
type CheckCallForSentReplayResult = {pass: boolean, call: Call|undefined, results: Result[]};
```

but whitespace/newlines would be good too

Reply:
Hmm yeah, not having precommit hooks :(